### DISCUSSION TITLE
Question about replication

### DISCUSSION THREAD

**Discussion by alpharde** [seaweedfs/seaweedfs#2988] [Q&A]
Sorry I could not find this anywhere on the docs, how do I set seaweedfs to create a copy of everything in each volume?

**Comment by chrislusf** [comment]
not clear what is the question.

**Comment by alpharde** [comment]
I'm testing seaweedfs as a kind of replacement for the famed zfs filesystem, in which the Z1 mode replicates the data in three disks with failure tolerance for one disk.
Since zfs doesn't have any sort of network capability right now, I would ask if seaweedfs could operate in a similar mode and what configuration should I use to that effect.

**Reply by olivier-norture** [reply]
I was looking for same kind of information. I hope you got you answer since 2022 ;)

This article answer everything;  I left it here for next ones:
https://github.com/seaweedfs/seaweedfs/wiki/Erasure-coding-for-warm-storage
